### ISSUE TITLE
Square bale loader just stops

### ISSUE THREAD

**Issue by SpiderTash** [Courseplay/courseplay#7014]
When using the new Mode 7 for loading square bales on the in game stacker, it might load 1-3 bales and just stop.  It is random on how many bales it will load before it stops and then sometimes it will finish off the rack and then go unload and start the problems again.  I did just update to revision 51 and try again and still an issue.  I know it worked fine on an early version (48 or 49) with the round bales but I did not try square bales on that version.
[log.txt](https://github.com/Courseplay/courseplay/files/6176846/log.txt)

**Comment by Tensuko** [comment]
You talk about the Forstack 8.12 ?

**Comment by SpiderTash** [comment]
I am actually using the FSX 63.72.  The equipment was borrowed through a contract, I have been fighting it for 2 days with saving and restarting the game and still the issue.  While typing this it loaded up 1 or 2 full racks just fine and then did it again after 1 bale.

**Comment by Tensuko** [comment]
logfile with debugchannel 7

**Comment by polosubtil** [comment]
Hi guys,

I have the same problem since the new reverse feature, it was working fine before. But this new feature is necessary as the odd shape fields makes it challenging. With version 49, the course would stop if pathfinding was failing. With version 50, it was already much better but pathinding would make everything lag until I would stop the course manually if the tractor was stuck in front of a bale or a hedge
In version 51, in most cases when I start the course in the field (using "collect bales" as a starting point), the tractor reverse right away and then stops the course when it stops reversing

Log with different attempts to make it work, I even tried to manually load a few bales
[log.txt](https://github.com/Courseplay/courseplay/files/6184983/log.txt)

**Comment by pvaiko** [comment]
You did not have the debug channel 11 on for pathfinding, so we don't know why the pathfinding failed. Would be great to have that info.

**Comment by polosubtil** [comment]
Oups sorry
Here you go : 
[log.txt](https://github.com/Courseplay/courseplay/files/6185958/log.txt)

It did it a few times to start with and then I drove manually to a group of bales and it started working when close enough and sort of aligned with the first bale

**Comment by pvaiko** [comment]
If you'd like, give it a try with the https://github.com/Courseplay/courseplay/tree/issue-7014 branch.

**Comment by polosubtil** [comment]
Hi @pvaiko 
I tried and it's already much better. If standing in front of a hedge, it reverse and get to work most of the time (sometimes it reverses and end work). But if standing in front of a bale, it just stays stuck and stop the work.
![image](https://user-images.githubusercontent.com/60451845/112216944-623eed80-8c22-11eb-964a-45e43ba73315.png)
[log.txt](https://github.com/Courseplay/courseplay/files/6192655/log.txt)

**Comment by polosubtil** [comment]
This one is starting in front of the hedge. It reverses and then stops
![image](https://user-images.githubusercontent.com/60451845/112217234-afbb5a80-8c22-11eb-9f38-b19c0a8536cb.png)
It should be the latest entry in the log
[log.txt](https://github.com/Courseplay/courseplay/files/6192672/log.txt)

**Comment by SpiderTash** [comment]
Got some bonus log here.  Had some bales wrapped with the SW 4014 and a couple bales already on the FSX 63.72 trailer.  I start the trailer and it picks up the rest of the bales to fill it fine and goes and unloads.  While it is going to unload I start the wrapper again, and if it was on its way to a bale but it picked up a different bale on the way there, it would get to the bale it intended to pick up, finish wrapping the bale it did pick up, drop it, and then also stop the CoursePlay, did that 4-5 times.
While the wrapper was still running now, when the trailer got back it picked up a wrapped bale and stopped, I started it again, it picked up two unwrapped bales and then stopped.
Sorry for the delay in getting this log to you.
[log.txt](https://github.com/Courseplay/courseplay/files/6193551/log.txt)

**Comment by polosubtil** [comment]
Works like a charm, great work as usual guys ;) Thanks !!!!!!!!!!!!

**Comment by SpiderTash** [comment]
Sorry I am late, I have been very busy and have not had time to play.  So I updated to .52 and have not tested the square bale loaded, but I think the issue I was having was different than what was found.  So I am having the same problem with the round bale loader (Ursus T127 Plus) that it would pick up two bales and then stop.  It seemed like it would happen if the bale wrapper (Kuhn SW 4014) was wrapping a bale.  I decided to just grab another bale wrapper and not pick them up till they were all wrapped.  Turns out that the 2nd bale wrapper would have the same problem.  If it finished wrapping a bale and the first bale wrapper was wrapping a bale, the 2nd bale wrapper would stop.
Is the programming only set up to have 1 pieces of equipment using the wrap/pickup mode at a time?
